src/librustc_codegen_llvm/llvm_util.rs

```diff
@@ -174,7 +174,7 @@ const X86_WHITELIST: &[(&str, Option<Symbol>)] = &[
 
 const HEXAGON_WHITELIST: &[(&str, Option<Symbol>)] = &[
     ("hvx", Some(sym::hexagon_target_feature)),
-    ("hvx-double", Some(sym::hexagon_target_feature)),
+    ("hvx-length128b", Some(sym::hexagon_target_feature)),
```

Comment:
This isn't strictly part of the `hexagon-unknown-linux-musl` target support, just found this mismatch while reviewing.  I can leave this part out of the change if that makes more sense.